eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -61,11 +61,12 @@ object Graph {
    * The fee for a failed attempt and the fee per hop are never actually spent, they are used to incentivize shorter
    * paths or path with higher success probability.
    *
-   * @param lockedFundsRisk cost of having funds locked in htlc in msat per msat per block
-   * @param failureCost     fee for a failed attempt
-   * @param hopCost         virtual fee per hop (how much we're willing to pay to make the route one hop shorter)
+   * @param lockedFundsRisk   cost of having funds locked in htlc in msat per msat per block
+   * @param failureCost       fee for a failed attempt
+   * @param hopCost           virtual fee per hop (how much we're willing to pay to make the route one hop shorter)
+   * @param usePastRelaysData use data from past relays to estimate the balance of the channels
    */
-  case class HeuristicsConstants(lockedFundsRisk: Double, failureCost: RelayFees, hopCost: RelayFees)
+  case class HeuristicsConstants(lockedFundsRisk: Double, failureCost: RelayFees, hopCost: RelayFees, usePastRelaysData: Boolean)
```

Comment:
Why isn't this called `useBalanceEstimates`? It would be more coherent, wouldn't it?